src/components/PostList.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Post } from '../types/Post';
+import { PostInfo } from './PostInfo';
+
+type Props = {
+  posts: Post[];
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => {
+  return (
+    <div className="posts-list">
+      {posts.map((post) => (
+        <div className="posts-list__post post">
+          <PostInfo
+            title={post.title}
+            body={post.body}
+            user={post.user}
+            comment={post.comment}
+          />
+        </div>
+      ))}
+    </div>
```

Comment:
Why not use `ul -> li`?

```suggestion
    <ul className="posts-list">
      {posts.map((post) => (
        <li className="posts-list__post post" key={post.id}>
          <PostInfo
            title={post.title}
            body={post.body}
            user={post.user}
            comment={post.comment}
          />
        </li>
      ))}
    </ul>
```